test/sim/moves/gmaxvolcalith.js

```diff
@@ -40,4 +40,34 @@ describe('G-Max Volcalith', function () {
 		battle.makeChoices('move sleeptalk, move sleeptalk', 'move sleeptalk, move sleeptalk');
 		assert.strictEqual(battle.p2.active[0].hp, battle.p2.active[0].maxhp - (Math.floor(battle.p2.active[0].maxhp / 6) * 4));
 	});
+
+	it.skip('should deal damage alongside Sea of Fire or G-Max Wildfire in the order those field effects were set', function () {
+		battle = common.createBattle({gameType: 'doubles'});
+		battle.setPlayer('p1', {team: [
+			{species: 'Coalossal-Gmax', item: 'Eject Button', moves: ['rockthrow', 'sleeptalk']},
+			{species: 'Charmander', moves: ['firepledge', 'sleeptalk']},
+			{species: 'Wynaut', level: 72, moves: ['grasspledge', 'nightshade', 'sleeptalk']},
+		]});
+		battle.setPlayer('p2', {team: [
+			{species: 'Roggenrola', ability: 'Receiver', item: 'Focus Sash', level: 1, moves: ['sleeptalk', 'tackle']},
+			{species: 'Boldore', ability: 'Receiver', item: 'Sitrus Berry', evs: {hp: 28}, moves: ['sleeptalk']},
+			{species: 'Torracat', ability: 'Cheek Pouch', item: 'Focus Sash', level: 1, moves: ['sleeptalk']},
+		]});
+		//Set up Volcalith first, then Sea of Fire
```

Comment:
```suggestion
		// Set up Volcalith first, then Sea of Fire
```

PS convention is to have spaces after `//`, if you could also make those changes to the other places.